mrbelvedereci/plan/models.py

```diff
@@ -45,13 +50,23 @@ class Plan(models.Model):
     yaml_config = models.TextField(null=True, blank=True, validators=[validate_yaml_field])
 
     class Meta:
-        ordering = ['name','repo__owner','repo__name', 'active', 'context']
+        ordering = ['name', 'active', 'context']
 
     def get_absolute_url(self):
         return reverse('plan_detail', kwargs={'plan_id': self.id})
 
     def __unicode__(self):
-        return u'[{}] {}'.format(self.repo.name, self.name)
+        return u'{}'.format(self.name)
+
+    @property
+    def repo_names(self):
+        names = [repo.name for repo in self.repos.all()]
+        return ', '.join(names)
+
+    @property
```

Comment:
I feel really skeezy having a property return a generator. I would feel more comfortable if it were a method call.